src/components/Forms/Category-form.tsx

```diff
@@ -53,8 +44,7 @@ const CategoryForm:React.FC<CategoryFormProps> = ({dispatch, categoriesLength, h
        onSubmit={(values, { setSubmitting }) => {
          setTimeout(() => {
```

Comment:
why is setTimeout needed?

Reply:
Not nedded. It's has been made for emulation slow network request.